### PR TITLE
Add proper licenses and references for Danish dictionary

### LINKED ISSUE #26: Problems with the danish dictionary
2 days ago, I submitted a danish dictionary.
It seems like something have gone wrong when compressing `da_DK.trie.gz`
The file is only 20 bytes large, and when I uncompress the file, only an empty file comes out. [Link to file](https://github.com/Jason3S/cspell-dicts/blob/master/packages/da_DK/da_DK.trie.gz).

I have tried to compress the .dic file using cspell-tools, but this also generates a `.trie.gz` with the size of 20 bytes.
```bash
$ cspell-tools compile-trie da_DK.dic

Compile:
 output: default
 compress: true
 files:
  da_DK.dic


Process "da_DK.dic" to "da_DK.trie.gz"
Complete.
```

I have tried to load the .dic file uncompressed in my user settings:

```json
"cSpell.dictionaryDefinitions": [
        {
            "name": "danish",
            "path": "/Users/viktorstrate/Documents/dictionaries/da_DK.dic",
            "description": "Danish dictionary"
        }
    ],
```

But it doesn't look like it reconizes the affix file (.aff) located at "/Users/viktorstrate/Documents/dictionaries/da_DK.aff". Do I have to specify this path in my user settings, for cspell to use it?

Currently I use a 4,5mb wordlist, this works fine, but the CPU running at 100% when loading the word list.

```json
"cSpell.dictionaryDefinitions": [
        {
            "name": "danish-wordlist",
            "path": "/Users/viktorstrate/Documents/dictionaries/da_DK.txt",
            "description": "Danish (txt wordlist)"
        }
    ],
```

EDIT:
It looks like this is a problem with `hunspell-reader`, as it cannot convert the dic and aff files. Jason3S/hunspell-reader#2

### PR DESCRIPTION
**Pull request by viktorstrate**
This is some final work to properly cite the original authors of the danish dictionary.
This closes #26.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -39,3 +39,6 @@ jspm_packages
 # Temp files
 tmp
 temp
+
+# Mac files
+.DS_Store
```
Comment (Jason3S) [inline .gitignore]: You might want to consider adding this to your global git settings.

File: packages/da_DK/LICENSE [hunk 2]
```diff
@@ -1,21 +1,12 @@
...
-all copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-THE SOFTWARE.
+Stavekontrolden - Danish dictionary files for Hunspell
```
Comment (Jason3S) [inline packages/da_DK/LICENSE]: There are two license, so we need to keep both.
There is one license for this package and a separate license for the hunspell files.

**Comment by viktorstrate** [comment]
I didn't know about global git ignores, thanks for the tip ;)
I have explained the licenses in LICENSE, and added the individual licenses, in each own file.

**Comment by Jason3S** [comment]
Thank you.
I'm still looking at the list of words that was generated from the hunspell. It doesn't seem complete. This might still be an issue with how the hunspell reader is working.

**Pull request merged**